### ISSUE TITLE
[Re] Learning Fair Graph Representations via Automated Data Augmentations

### ISSUE THREAD

**Issue by Thiggel** [ReScience/submissions#90]
**Original article:**

**PDF URL:**  https://github.com/Thiggel/FACT/blob/main/Paper.pdf
**Metadata URL:**  https://github.com/Thiggel/FACT/blob/main/metadata.yaml
**Code URL:**  https://github.com/Thiggel/FACT

**Scientific domain:**  Artificial Intelligence
**Programming language:**  Python
**Suggested editor:**

**Comment by rougier** [comment]
Thanks for your submission. Was your paper part of the ICLM reproducibility challenge na dif es, are there any open peer review available ?

**Comment by Thiggel** [comment]
Yes it was, the review can be found here: https://openreview.net/forum?id=9OlU865dAF

**Comment by rougier** [comment]
Oh nice, so we don't need to review again. What is not clear to me is the reason TMLR rejected your paper in the first place. Anyway, based on the reviews and the revisions you made, I think we can publish it provided you use the ReScience template. Can you adapt the paper and possibly put it in a dedicated repo (or directory in your project) since I'll need to fork it for publication.

**Comment by MaxBelitsky** [comment]
Hi @rougier, we added a directory with the ReScience template: https://github.com/Thiggel/FACT/tree/main/rescience_submission. Please, let us know if anything else is needed.

**Comment by rougier** [comment]
Got it. Can you save your Github repo on Software Heritage and post here the swid you'll get?

**Comment by Thiggel** [comment]
Alright, the id is 417230531. Let us know if you need anything else.

**Comment by rougier** [comment]
Thanks. It shoud start with `swid`. Can you post the full id?

**Comment by Thiggel** [comment]
The origin SWHID is:
swh:1:ori:35c0a91c31b06a24561bece63e76ccecc1787bcb

The snapshot SWHID is:
swh:1:snp:894decb239b454eaed7dc5c99d9fe6b5400d7a26

Is this what you mean? or where can I find the swid?

**Comment by rougier** [comment]
Sorry I'm late publishing your paper. Can you give ORCID for all authors?

**Comment by Thiggel** [comment]
Hey, the orcids are as follows:

Max Belitsky: 0009-0000-0396-2916
Nordin Belkacemi: 0009-0005-0288-3024
Filipe Laitenberger: 0009-0003-7836-9828
Denys Sheremet does not have an ORCID id.

Best regards

**Comment by rougier** [comment]
Thanks. The **sandbox** version of your article is [here](https://sandbox.zenodo.org/record/295008). Can you check everything is ok (especially all links). Once check I'll publish the actual version (with a different DOI)

**Comment by MaxBelitsky** [comment]
Hi @rougier, we reviewed the sandbox version and found some things that need to be changed:
- The abstract is incorrect – it is from the paper we reproduced, not from our paper
- An indication of equal contribution of all authors is missing
- A comma after the 3rd author's name is missing
- The link to our codebase on page 2 leads to an anonymous repo (we forgot to update it)

Other than that, everything seems to be fine. All the links work.

How can we make the changes? Is there a branch you used that we can create a PR to?

**Comment by rougier** [comment]
- Weird, I cloned your repo and use your rescience_submission. Did you put your abstract in the matadata.yaml file ? Where is the right abstract then?
- For equal contribution, how does it appear on the paper? Did ou sue special signs?
- I don't quite understand why it is the case.
- Can you give me the right repo then?

**Comment by MaxBelitsky** [comment]
@rougier 

We had problems compiling locally as the script broke, so we used Overleaf and modified the `metadata.tex` manually instead and forgot to modify the YAML file. I've made the following changes to our repo:

- I updated the `metadata.yml` in our repo with the correct abstract
- For equal contribution, I added another affiliation in `metadata.yml`. I hope this would work. When we compiled the document, the `metadata.tex` looked like this:
```
...
\author[1, *]{Max Belitsky}
\author[1, *]{Filipe Laitenberger}
\author[1, *]{Denys Sheremet}
\author[1, *]{Nordin Belkacemi}
\affil[*]{Equal contribution}
\affil[1]{University of Amsterdam, Amsterdam, Netherlands}
```
I added the same symbol in `metadata.yml`. Hopefully, the script would produce the same tex code.

```
authors:
  - name: Max Belitsky
    orcid:
    email: max.belitsky@student.uva.nl
    affiliations: 1,*
    
  - name: Filipe Laitenberger
    orcid:
    email: filipe.laitenberger@student.uva.nl
    affiliations: 1,*

  - name: Denys Sheremet
    orcid: 
    email: denys.sheremet@student.uva.nl
    affiliations: 1,*

  - name: Nordin Belkacemi
    orcid:
    email: nordin.belkacemi@student.uva.nl
    affiliations: 1,*

affiliations:
  - code:    1
    name:    University of Amsterdam
    address: Amsterdam, Netherlands
  - code:    "*"
    name:    Equal contribution
```

- I guess that's not important, we can leave it like this if there is no easy fix
- Fixed the URL in the footnote in our repo

**Comment by rougier** [comment]
Thanks. I'm almost done. Only the missing , to fix but I've no idea why it is missing. But I want to know why.

**Comment by rougier** [comment]
It's online at https://zenodo.org/records/16374814. Congratulations. It will soon appear on ReScience website.
